configs/trip_mode_choice.csv

```diff
@@ -1,405 +1,405 @@
+DRIVEALONEPAY - Unavailable for zero auto households,auto_ownership == 0,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable for persons less than 16,age < 16,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable for joint tours,is_joint == True,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Unavailable if didn't drive to work,is_atwork_subtour & ~work_tour_is_SOV,,-999,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - In-vehicle time,@c_ivt*odt_skims['SOVTOLL_TIME'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Terminal time,@c_walktimeshort * df.total_terminal_time,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Operating cost,@df.c_cost * costPerMile * odt_skims['SOVTOLL_DIST'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Parking cost,c_cost * total_parking_cost,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Bridge toll,@df.c_cost * odt_skims['SOVTOLL_BTOLL'],,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Value toll,@df.c_cost * odt_skims['SOVTOLL_VTOLL'] + cordon_toll  * df.is_dest_cordon - income_low_100_discount * df.is_income_lower_100,,1,,,,,,,,,,,,,,,,,,,
+DRIVEALONEPAY - Person is between 16 and 19 years old,@c_age1619_da * ((df.age >= 16) & (df.age <= 19)),,1,,,,,,,,,,,,,,,,,,,
+#Shared ride 2,,,,,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Unavailable,hov2_available == False,,,-999,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Unavailable based on party size,is_joint & (number_of_participants > 2),,,-999,,,,,,,,,,,,,,,,,,
+SHARED2FREE - In-vehicle time,@c_ivt * odt_skims['HOV2_TIME'],,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Terminal time,@c_walktimeshort * df.total_terminal_time,,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Operating cost,@df.c_cost * costPerMile * odt_skims['HOV2_DIST'],,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Parking cost,@df.c_cost * df.total_parking_cost / costShareSr2,,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Bridge toll,@df.c_cost * odt_skims['HOV2_BTOLL'] / costShareSr2,,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - One person household,@c_hhsize1_sr * (df.hhsize == 1),,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Two person household,@c_hhsize2_sr * (df.hhsize == 2),,,1,,,,,,,,,,,,,,,,,,
+SHARED2FREE - Person is 16 years old or older,@c_age16p_sr * (df.age >= 16),,,1,,,,,,,,,,,,,,,,,,
+#Shared ride 2 toll,,,,,,,,,,,,,,,,,,,,,,
+SHARED2PAY - Unavailable,hov2toll_available == False,,,,-999,,,,,,,,,,,,,,,,,
+SHARED2PAY - Unavailable based on party size,is_joint & (number_of_participants > 2),,,,-999,,,,,,,,,,,,,,,,,
+SHARED2PAY - In-vehicle time,@c_ivt * odt_skims['HOV2TOLL_TIME'],,,,1,,,,,,,,,,,,,,,,,
+SHARED2PAY - Terminal time,@c_walktimeshort * df.total_terminal_time,,,,1,,,,,,,,,,,,,,,,,
+SHARED2PAY - Operating cost,@df.c_cost * costPerMile * odt_skims['HOV2TOLL_DIST'],,,,1,,,,,,,,,,,,,,,,,
+SHARED2PAY - Parking cost,@df.c_cost * df.total_parking_cost / costShareSr2,,,,1,,,,,,,,,,,,,,,,,
+SHARED2PAY - Bridge toll,@df.c_cost * odt_skims['HOV2TOLL_BTOLL'] / costShareSr2,,,,1,,,,,,,,,,,,,,,,,
+SHARED2PAY - Value toll,@df.c_cost * odt_skims['HOV2TOLL_VTOLL'] / costShareSr2,,,,1,,,,,,,,,,,,,,,,,

```

Comment:
I think we'd also want to add the cordon toll here (and for SHARED3PAY)

Reply:
That's right, done !